NEXT_CHANGELOG.md

```diff
@@ -24,124 +24,9 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 -->
 
 # [x.x.x] (unreleased) - 2022-mm-dd
-
+> Important: X breaking changes below, indicated by **❗ BREAKING ❗**
```

Comment:
Is this a new addition to the NEXT_CHANGELOG.md?

Reply:
i just copied it from the top.  we can remove it.